docs/adrs/003-bugzilla-response-codes.md

```diff
@@ -0,0 +1,100 @@
+# Bugzilla disables webhooks after too many errors
+
+- Status: proposed
+- Date: 2023-27-11
+
+Tracking issue: 
+
+## Context and Problem Statement
+When bugzilla receives too many error responses from JBI, it stops triggering webhook calls for the entire project causing data to stop syncing. Frequently these errors are due to JBI being unable to process a payload due to errors in configuration (or incomplete configuration) in Jira or a mismatch of data for a single bug. 
```

Comment:
The most recent incidents have been because the webhook payload doesn't match our schemas in some way. FastAPI will return `422` errors for Pydantic validation errors.